webapi/src/main/scala/org/knora/webapi/routing/admin/UsersRouteADM.scala

```diff
@@ -156,6 +156,10 @@ class UsersRouteADM(_system: ActorSystem, settings: SettingsImpl, log: LoggingAd
 
                             val userIri = stringFormatter.validateAndEscapeIri(value, throw BadRequestException(s"Invalid user IRI $value"))
 
+                            if (userIri.equals(KnoraSystemInstances.Users.SystemUser.id) || userIri.equals(KnoraSystemInstances.Users.AnonymousUser.id)) {
+                                throw BadRequestException("Changes to built-in users are not allowed.")
```

Comment:
Should be `ForbiddenException`?

Reply:
Yea, I was debating this with myself. `Forbidden` would imply that the calling user does not have the permission but that somebody else has. In this case, nobody will ever be allowed to perform this operation and that is why I'm treating it like a `BadRequest`.

But you decide ;-)

Reply:
I see what you mean. I think either would make sense. So let's leave it like this, it's easier. :)

Reply:
:-)